intel_extension_for_transformers/llm/runtime/graph/application/main_run.cpp

```diff
@@ -654,7 +666,16 @@ int main(int argc, char** argv) {
     }
 
     // end of text token
-    if (!embd.empty() && embd.back() == model_token_eos()) {
+    if (params.model_arch == MODEL_CHATGLM) {
+      if (!embd.empty() && embd.back() == 130005) {
```

Comment:
I'd prefer to set 130005 as a static explicit variable

Reply:
fixed